src/renderer/store/modules/session.js

```diff
@@ -186,6 +194,11 @@ export default {
       const profile = rootGetters['profile/byId'](profileId)
       if (!profile) return
 
+      if (!profile.unconfirmedVotes) {
+        profile.unconfirmedVotes = []
+        dispatch('profile/update', profile)
```

Comment:
`[vuex] unknown local action type: profile/update, global type: session/profile/update` on the dashboard